V3.bug fixing

kelscahill opened the pull request:
Closes #288, #283 

I renamed the `Story Block` to `Content Block Expand` which I think makes a little more sense. Let me know if this works. Also, there was no design for the Testimonies organism, so I just kept it as a carousel, but used the grid layout.

designerbrent commented:
What about putting them in the top right corner? 

![2018-03-20 at 7 22 pm](https://user-images.githubusercontent.com/132864/37688096-aa86d65e-2c74-11e8-838f-8c27763e5fc4.png)

ruiz-knott commented:
The button component should have a bit of padding on the left side to align with the text of the testimonies.

kelscahill commented:
@designerbrent I added the testimonies carousel arrows, however, I didn't include the counter at this time since that is quite technical to implement. Since we are low on hours, I fixed other issues in the list.

Closes #293, #290, #292

designerbrent commented:
These look great, @kelscahill. Not having the counter on the testimonies is fine.

designerbrent commented:
Once you are good with this @tpitre, let's commit it in.
